tests/py/dynamo/models/test_export_serde.py

```diff
@@ -82,6 +82,9 @@ def test_base_full_compile_multiple_outputs(ir):
     with multiple outputs which is fully TRT convertible
     """
 
+    tmp_dir = tempfile.mkdtemp(prefix="test_base_full_compile_multiple_outputs")
```

Comment:
I think pytest has something we could use here if you want to look into it. Can give a temp dir as argument

Reply:
https://docs.pytest.org/en/6.2.x/tmpdir.html